src/app/modules/main/wallet_section/filter.nim

```diff
@@ -25,10 +25,9 @@ proc `$`*(self: Filter): string =
     chainIds: {self.chainIds},
     )"""
 
-proc setFillterAllAddresses*(self: Filter) = 
+proc setFillterAllAddresses*(self: Filter) =
   self.allAddresses = true
-
-  self.addresses = self.controller.getWalletAccounts().filter(a => not a.hideFromTotalBalance).map(a => a.address)
+  self.addresses = self.controller.getWalletAccounts().map(a => a.address)
```

Comment:
That probably impact the total balance no?

Reply:
it does impact the total balance on wallet view in the header unfortunately.